src/ansys/aedt/core/emit_core/nodes/generated/Amplifier.py

```diff
@@ -0,0 +1,255 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 - 2025 ANSYS, Inc. and/or its affiliates.
+# SPDX-FileCopyrightText: 2021 - 2025 ANSYS, Inc. and /or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+from enum import Enum
+
+from ..EmitNode import EmitNode
+
+
+class Amplifier(EmitNode):
+    def __init__(self, emit_obj, result_id, node_id):
+        self._is_component = True
+        EmitNode.__init__(self, emit_obj, result_id, node_id)
+
+    def rename(self, new_name: str):
+        """Rename this node"""
+        self._rename(new_name)
+
+    def duplicate(self, new_name: str):
+        """Duplicate this node"""
+        return self._duplicate(new_name)
+
+    def delete(self):
+        """Delete this node"""
+        self._delete()
+
+    @property
+    def filename(self) -> str:
+        """Filename
+        Name of file defining the outboard component
```

Comment:
```suggestion
        """Name of the file defining the outboard component.
```
Could you perform the same kind of changes in other methods to have a simple and short title description ending with a dot ?

Reply:
@SMoraisAnsys Have now reformatted all the docstrings, every property's first sentence should either be a title or short description, both ending in a period.